lib/otel/setup.js

```diff
@@ -24,24 +25,20 @@ module.exports = function setupOtel(agent, logger = defaultLogger) {
 
   createOtelLogger(logger, agent.config)
 
-  const provider = new BasicTracerProvider({
+  opentelemetry.trace.setGlobalTracerProvider(new BasicTracerProvider({
```

Comment:
this was done in prep for the removal of the register method, [see](https://github.com/open-telemetry/opentelemetry-js/commit/544c40984797dd2e2cccb92cce0e88b30f235b02)